src/config.ts

```diff
@@ -20,28 +20,58 @@ export interface ValidationConfig {
   readonly properties?: { [key: string]: any };
 }
 
+export enum SynthesisFormat {
+  'CDK8s' = 'cdk8s',
+  'HELM' = 'helm',
+}
+
+export enum HelmChartApiVersion {
+  'V1' = 'v1',
+  'V2' = 'v2',
+}
+
+export interface HelmSynthesis {
+  readonly chartApiVersion?: HelmChartApiVersion;
+  readonly chartVersion?: string;
+}
+
 export interface Config {
   readonly app?: string;
   readonly language?: Language;
   readonly output?: string;
   readonly imports?: string[];
   readonly pluginsDirectory?: string;
   readonly validations?: string | ValidationConfig[];
+  readonly format?: SynthesisFormat;
+  readonly helmSynthConfig?: HelmSynthesis;
 }
 
 const DEFAULTS: Config = {
   output: 'dist',
   pluginsDirectory: path.join(os.homedir(), '.cdk8s', 'plugins'),
+  format: SynthesisFormat.CDK8s,
 };
 
 export function readConfigSync(): Config {
   let config: Config = DEFAULTS;
+
   if (fs.existsSync(CONFIG_FILE)) {
     config = {
       ...config,
       ...yaml.parse(fs.readFileSync(CONFIG_FILE, 'utf-8')),
     };
   }
+
+  if (config.format === SynthesisFormat.HELM && config.helmSynthConfig && !config.helmSynthConfig.chartApiVersion) {
+    config = {
+      ...config,
+      helmSynthConfig: {
+        ...config.helmSynthConfig,
+        chartApiVersion: HelmChartApiVersion.V2,
+      },
+    };
+  }
```

Comment:
This looks like you're setting the default value of `chartApiVersion`, but you're only doing so if `config.helmSynthConfig` is present - why? The most common usage would actually be when the config file doesn't include this, and the `chart-version` is passed via command line. Also note that your default value actually overrides any user defined one...

```suggestion
  if (config.format === SynthesisFormat.HELM) {
    config = {
      ...config,
      helmSynthConfig: {
        chartApiVersion: HelmChartApiVersion.V2,
        ...(config.helmSynthConfig ?? {}),
      },
    };
  }
```